src/ui/base-page-storyEdit.js

```diff
@@ -35,6 +35,16 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.
                     }
                 },
                 target: "{that storyEditor blockManager}.options.modelListeners"
+            },
+            "story.createNewStoryOnServer": {
+                record: {
+                    "": {
```

Comment:
With `createNewStoryOnServer` triggered by the model change to the root path of the `story` component, it means every story content change will also trigger this listener, which might be unnecessary. I wonder if it's possible to use `includeSource: "init"` to only trigger this listener at the component initialization?

Reply:
`init` is too early for this to fire, since that is effectively the same as having it call `createNewStoryOnServer` on page load, which is not what we're aiming for at the moment since we want to avoid clogging up the database with story records.

Currently, the function that's called by the invoker checks whether there's an `id` key on the story model, which is a workable solution in the "short" term, but ideally we would have a no-op for changes after the "first" one. It's an interesting challenge.

@jobara and @amb26 had a discussion in Matrix last week (see "Fluid Technology" from 2021-02-17), and @amb26's suggestion of using a flag to track the first model change and attaching a listener to that flag seems like a promising solution to this awkwardness:

> Just create a model listener which listens to "" and produces a {changePath: "userInteractionReceived", value: true} and then attach the listener which checks whether there is an id to that flag

Reply:
I see. Thanks for the explanation, @BlueSlug.